src/main/resources/application.conf

```diff
@@ -12,65 +12,29 @@ kamon {
 
   show-aspectj-missing-warning = on
 
-  metric {
-    filters {
```

Comment:
What happens with all these filters in Kamon 1.x?  They are simply not needed?

Reply:
Filters are configure using `kamon.util.filters` property https://kamon.io/documentation/1.x/recipes/migrating-from-kamon-0.6/#filters-are-configured-differently